db/migrate/20171006145557_create_restaurants.rb

```diff
@@ -3,6 +3,7 @@ def change
     create_table :restaurants do |t|
       t.string :name
       t.string :address
+      t.belongs_to :restaurant_category, index: true
```

Comment:
There's a danger in modifying an existing migration. Will probably cause failure on deployment. Create new migrations instead to avoid conflicts